Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM starefossen/github-pages
+FROM starefossen/github-pages:112
```

Comment:
Besides just stability and ensuring we ship a site that actually builds, these:

- New plugin: jekyll-include-cache
- Bump jekyll-titles-from-headings to v0.1.3
- Bump jekyll-readme-index to v0.0.3
- The following enabled-by-default plugins (+ a default theme):

Jekyll Optional Front Matter - Don't require optional front matter to process a page as a page
Jekyll README Index - If a repo has a readme, but not an index, use the readme as the index
Jekyll Default Layout - Set a default per-page layout if no layout is specified and a logical one exists for the site
Jekyll Titles from Headings - If there is no YAML title, use the first heading as the page's title
- Jekyll Relative Links